pkg/converter/target/externalnode.go

```diff
@@ -58,15 +60,16 @@ func NewExternalNodeFrom(
 
 // PatchExternalNodeFrom generate a patch for existing ExternalNode from source.
 func PatchExternalNodeFrom(
-	source ExternalNodeSource, patch *antreav1alpha1.ExternalNode, cl client.Client) *antreav1alpha1.ExternalNode {
-	populateExternalNodeFrom(source, patch, cl)
-	return patch
+	source ExternalNodeSource, patch *antreav1alpha1.ExternalNode, cl client.Client) (*antreav1alpha1.ExternalNode, bool) {
+	changed := populateExternalNodeFrom(source, patch, cl)
+	return patch, changed
 }
 
-func populateExternalNodeFrom(source ExternalNodeSource, externalNode *antreav1alpha1.ExternalNode, cl client.Client) {
+func populateExternalNodeFrom(source ExternalNodeSource, externalNode *antreav1alpha1.ExternalNode, cl client.Client) bool {
 	externalNode.SetLabels(genTargetEntityLabels(source, cl))
```

Comment:
When the VirtualMachineLabel gets changed, we want to update the ExternalEntity labels as well.
Please check if the labels are changed or not.

Reply:
changed